src/v_video.c

```diff
@@ -305,13 +308,7 @@ void V_DrawPatch(int x, int y, patch_t *patch)
 
 void V_DrawPatchFullScreen(patch_t *patch, boolean flipped)
 {
-    const short width = SHORT(patch->width);
-    const short height = SHORT(patch->height);
-
-    dx = (NONWIDEWIDTH << FRACBITS) / ORIGWIDTH;
```

Comment:
What will now happen to graphics larger than 320x200 px in non-widescreen mode? For example, the TITLEPIC of Harmony - not that it was supported anyway.

Reply:
If wider than 320, they'll be centered horizontally, with the parts that don't fit on screen clipped out.

I haven't tested taller than 200, but I'd guess they'd just be clipped.